db/data.py

```diff
@@ -58,6 +58,44 @@ def _has_key(dictionary, key):
         dictionary = dictionary[part]
     return True
 
+def check_highlevel_meta():
```

Comment:
This method name should more accurately describe what it does. Something like `get_highlevel_meta_count`. It should include a docstring that explains what it does and what the return value is.
Because this method explicitly returns a count, you should read this value from the query result and return a number, instead of having to do `h[0][0]` in the management command.
This method should also have a test to ensure that it operates as expected.

Reply:
i have removed check_highlevel and check_highlevel_meta functions and merged the functionality to one single function show_failed_rows which returns the list of failed rows and with their (id,mbid and offset)